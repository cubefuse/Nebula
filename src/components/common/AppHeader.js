import React from "react";
import { Layout } from "antd";

const { Header } = Layout;

/**
 * App Header component. Contains the top bar.
 */
export default class AppHeader extends React.Component {
  render() {
    return (
      <Header className="header">
        <div style={{ color: "white" }}>
          <span style={{ fontSize: "x-large", paddingRight: "5px" }}>
            <strong>Nebula</strong>
          </span>
          <span style={{ fontSize: "large", paddingRight: "10px" }}>
            Web UI
          </span>
          <span style={{ fontSize: "small" }}>POC-01</span>
        </div>
      </Header>
    );
  }
}
