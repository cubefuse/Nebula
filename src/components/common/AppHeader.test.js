import React from "react";
import "jest-dom/extend-expect";
import { render } from "react-testing-library";
import AppHeader from "./AppHeader";

test("can render", () => {
  const node = render(<AppHeader />);
  expect(node).toMatchSnapshot();
});
