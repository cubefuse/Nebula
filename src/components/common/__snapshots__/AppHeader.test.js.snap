// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can render 1`] = `
Object {
  "container": <div>
    <div
      class="header ant-layout-header"
    >
      <div
        style="color: white;"
      >
        <span
          style="font-size: x-large; padding-right: 5px;"
        >
          <strong>
            Nebula
          </strong>
        </span>
        <span
          style="font-size: large; padding-right: 10px;"
        >
          Web UI
        </span>
        <span
          style="font-size: small;"
        >
          POC-01
        </span>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "getAllByAltText": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getAllByValue": [Function],
  "getByAltText": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "getByValue": [Function],
  "queryAllByAltText": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryAllByValue": [Function],
  "queryByAltText": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "queryByValue": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
